conda-recipe/meta.yaml

```
@@ -51,7 +51,7 @@ requirements:
         - sphinxcontrib-bibtex
         - curio >=1.0
         - pineappl >=0.5.8
-        - eko >=0.12.2
+        - eko >=0.13.2
```

```suggestion
        - eko >=0.13.2,<0.14
```

To make sure we are safe from future big changes.

---

😅